gulpfile.js

```diff
@@ -337,6 +338,24 @@ gulp.task('bundle', ['buildnumber'], function () {
   return createBundle(DEFINES).pipe(gulp.dest(BUILD_DIR));
 });
 
+gulp.task('publish', ['generic'], function() {
```

Comment:
we need to return something (e.g stream) or make the function async. With async you may need to chain streams using `on('end', ...)` (see 'buildnumber' task). With pure parallel solution you can return streamqueue of "version.json" and "zip" pipes from the task.